sdk/appconfiguration/Azure.Data.AppConfiguration/src/Generated/ConfigurationClientBuilderExtensions.cs

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using Azure.Core.Extensions;
+using Azure.Data.AppConfiguration;
+
+namespace Microsoft.Extensions.Azure
+{
+    /// <summary> Extension methods to add <see cref="ConfigurationClient"/> to client builder. </summary>
+    public static partial class ConfigurationClientBuilderExtensions
+    {
+        /// <summary> Registers a <see cref="ConfigurationClient"/> instance. </summary>
+        /// <param name="builder"> The builder to register with. </param>
+        /// <param name="endpoint"> The endpoint of the App Configuration instance to send requests to. </param>
+        /// <param name="syncToken"> Used to guarantee real-time consistency between requests. </param>
+        public static IAzureClientBuilder<ConfigurationClient, ConfigurationClientOptions> AddConfigurationClient<TBuilder>(this TBuilder builder, Uri endpoint, string syncToken)
```

Comment:
This doesn't appear to be a constructor in the [public API](https://apiview.dev/Assemblies/Review/8b7f5312697a458ab9e65c2fd9cdc2dd?revisionId=a2869f4f8c184210b402803879b75bc8&doc=False#Azure.Data.AppConfiguration.ConfigurationSetting).   Where is `syncToken` coming from?